src/main/resources/swagger/agora.json

```diff
@@ -1,1353 +1,1442 @@
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "snapshotId",
+                        "in": "query",
+                        "description": "Snapshot ID of method.",
+                        "required": false,
+                        "type": "number"
+                    },
+                    {
+                        "name": "synopsis",
+                        "in": "query",
+                        "description": "Synopsis of method.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "documentation",
+                        "in": "query",
+                        "description": "Documentation of method.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "owner",
+                        "in": "query",
+                        "description": "Owner of method.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "payload",
+                        "in": "query",
+                        "description": "Payload of method in WDL.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "entityType",
+                        "in": "query",
+                        "description": "Type of Method - Task or Workflow",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "excludedFields",
+                        "in": "query",
+                        "description": "Fields to exclude in JSON response.",
+                        "required": false,
+                        "type": "string"
+                    },
+                    {
+                        "name": "includedFields",
+                        "in": "query",
+                        "description": "Fields to include in JSON response",
+                        "required": false,
+                        "type": "string"
+                    }
+                ],
+                "tags": [
+                    "Methods"
+                ],
+                "responses": {
+                    "200": {
+                        "description": "An array of methods.",
+                        "schema": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/MethodResponse"
+                            }
+                        }
+                    },
+                    "default": {
```

Comment:
Shouldn't this be "500"?  Currently its the only thing displayed under "Response Messages"